mm2src/lp_swap.rs

```diff
@@ -728,15 +729,57 @@ pub async fn insert_new_swap_to_db(
         .map_err(|e| ERRL!("{}", e))
 }
 
+async fn update_my_swap_coins_price_to_db(
+    ctx: MmArc,
+    uuid: Uuid,
+    my_coin_usd_price: &BigDecimal,
+    other_coin_usd_price: &BigDecimal,
+) -> Result<(), String> {
+    MySwapsStorage::new(ctx)
+        .update_coins_price(uuid, my_coin_usd_price, other_coin_usd_price)
+        .await
+        .map_err(|e| ERRL!("{}", e))
+}
+
+#[cfg(not(target_arch = "wasm32"))]
+async fn process_coins_price_update(ctx: &MmArc, swap: &SavedSwap) {
+    use super::lp_price::fetch_swap_coins_price;
+
+    if swap.is_finished_and_success() {
+        let cex_rates = fetch_swap_coins_price(
+            swap.maker_coin_ticker().unwrap_or_else(|_| "".to_string()).as_str(),
+            swap.taker_coin_ticker().unwrap_or_else(|_| "".to_string()).as_str(),
+        )
+        .await;
+        match cex_rates {
+            Some(response) => {
+                if let Err(e) =
+                    update_my_swap_coins_price_to_db(ctx.clone(), swap.uuid().to_owned(), &response.base, &response.rel)
```

Comment:
`process_coins_price_update` is used when a seednode receives `SavedSwap` that is needed to be written to the `stats_swaps` table. But why `process_coins_price_update` actually update prices in the `my_swaps` table?

Reply:
It seems that this comment is not related to the `process_coins_price_update` function implementation.
It seems that you forgot to remove `process_coins_price_update` calling right after `save_stats_swap`:
https://github.com/KomodoPlatform/atomicDEX-API/blob/27dea2752202c5d0dcb71edede86005312761783/mm2src/lp_swap.rs#L217-L220